src/components/ContactForm/FormContainer.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import Input from './Input';
+import TextArea from './TextArea';
+import Button from './Button';
+import './FormContainer.css';
+import * as emailjs from 'emailjs-com';
+
+class FormContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newUser: {
+        name: '',
+        senderEmail: '',
+        message: ''
+      },
+      sendingDisabled: true
+    };
+  }
+
+  componentDidMount () {
+    emailjs.init("user_zQzSvf211by5lDKI3R5pl");
+  };
+
+  handleFormSubmit = (event) => {
+    event.preventDefault();
+    const template = "template_0qVgU5Lv";
+
+    this.sendEmail(
+      template,
+      this.state.newUser.senderEmail,
+      this.state.newUser.name,
+      "Steven",
+      this.state.newUser.message
+      );
+
+    this.setState({
+      formSubmitted: true
+    })
+  };
+
+  sendEmail (templateId, senderEmail, fromName, toName, message) {
+    const { toastManager } = this.props;
+    this.setState({ sendingDisabled: true });
+    emailjs.send(
+'gmail',
+      templateId,
+      {
+        sender_email: senderEmail,
+        from_name: fromName,
+        to_name: toName,
+        message_html: message
+      })
+      .then(res => {
+        this.setState({ formEmailSent: true });
+        toastManager.add("Message sent successfully!", { appearance: 'success', autoDismiss: true });
+        this.handleClearForm();
+      })
+      .catch(err => {
+        console.error('Failed to send email, error: ', err);
+        toastManager.add("Sorry, there was a problem. Please try again!", { appearance: 'error', autoDismiss: true });
+      }
+    )
+  }
+
+  isSendingDisabled = () => {
+    return !!(Object.values(this.state.newUser).some(value => value === ''))
+  };
+
+  handleInput = e => {
+    const value = e.target.value;
+    const name = e.target.name;
+
+    this.setState(prevState => {
+      return {
+        newUser: {...prevState.newUser, [name]: value},
+        sendingDisabled: this.isSendingDisabled()
+      }
+    });
+  };
+
+  handleClearForm = () => {
+    this.setState({ newUser: {
+        name: '',
+        senderEmail: '',
+        message: ''
+    }})
```

Comment:
One way to tidy this up is to declare a constant outside of this class - e.g. BLANK_STATE - and set state equal to this variable where required (in constructor and here).